cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/ProgramOptionConstants.java

```diff
@@ -36,4 +36,6 @@
   public static final String WORKFLOW_BATCH = "workflowBatch";
 
   public static final String SCHEDULE_NAME = "scheduleName";
+
+  public static final String CONCURRENT_PROGRAM_RUN = "concurrent.program.run";
```

Comment:
It sounds like it should be in the program preference rather than a runtime arg that need to pass in every time.